client/components/breadcrumb/index.tsx

```diff
@@ -3,34 +3,47 @@ import styled from '@emotion/styled';
 import { useTranslate } from 'i18n-calypso';
 import { Key } from 'react';
 
+const flexAligned = {
+	display: 'flex',
+	alignItems: 'center',
+};
+
 const StyledUl = styled.ul`
-	display: flex;
-	align-items: center;
+	${ flexAligned };
 	list-style-type: none;
 	margin: 0;
 `;
 
 const StyledLi = styled.li`
-	display: flex;
-	align-items: center;
+	${ flexAligned };
 	font-size: 13px;
-	font-weight: 500;
-	--color-link: var( --studio-gray-100 );
+	font-weight: 400;
+	--color-link: var( --studio-gray-50 );
 
-	:first-of-type {
-		font-size: 16px;
-		font-weight: 600;
+	:last-of-type:not( :first-of-type ) {
 		--color-link: var( --studio-gray-80 );
+		font-weight: 500;
 	}
+`;
 
-	:last-of-type:not( :first-of-type ) {
-		--color-link: var( --studio-gray-50 );
+const StyledBackLink = styled.a`
+	${ flexAligned };
+	color: var( --studio-gray-80 ) !important; // It uses --studio-gray-50 if not using important
```

Comment:
Can we remove the `!important` in favor of

```
&:visited {
		color: var( --studio-gray-80 );
	}
```

Reply:
That will not display the color until the user visits the link, is that what we want here?

Reply:
I think we just have to override other pseudo-classes of the `a` element. This should work because the `a` element with the styled components class will have priority based on specificity:
```suggestion
	&,
	&:hover,
	&:focus,
	&:visited {
		color: var( --studio-gray-80 );
	}
```